src/index.js

```diff
@@ -1,18 +1,9 @@
 import Exporter from './exporter';
 import createTemplate from './template';
 
-let sql;
-
-if (process.env.APP_ENV === 'browser' || typeof window !== 'undefined') {
-  require('script-loader!sql.js');
-  sql = window.SQL;
-} else {
-  sql = require('sql.js');
-}
-
 export { Exporter };
 
-export default function(deckName, template) {
+export default function(deckName, template, sql) {
```

Comment:
What do you think about providing parameters via object? 
`export default function({deckName, template, sql}) {`
We now have 3 params, and 1st and 3rd are required, which is a little odd.

Also, should we provide a default value for `template`?

Reply:
I agree with @yakhinvadim, this does seem awkward as `template` is optional. I do see an advantage for using object notation for method params. This could be a good pattern that other methods could benefit from, e.g. `addCard`.

Reply:
Initially I didn't want to introduce breaking changes, but it totally makes sense. Thanks for insisting :+1: